woocommerce/woocommerce
# Replace Congratulations view Buttons with Links

Closes #51575 .



### How to test the changes in this Pull Request:



1. If needed, restart the Customize Your Store flow by installing the WooCommerce Beta Tester plugin and going to `wp-admin/tools.php?page=woocommerce-admin-test-helper > Tools > "Reset Customize Your Store"`.
2. In the admin, go to `WooCommerce > Customize Your Store > Create your own theme`.
3. Finish customizing and you will be redirected to the **Congratulations view**.
4. Try clicking on View store, Go to Products, Go to the Editor or Back to Home with your mouse middle click or via Ctrl+Click to open the links in a new tab.
5. Notice they don't open at all, because they are buttons.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Replaced Congratulations view Buttons with Links for Accessibility reasons.
</details>
